odoo/fields.py

```diff
@@ -789,6 +789,9 @@ def resolve_depends(self, registry):
                 try:
                     field = Model._fields[fname]
                 except KeyError:
+                    if registry.registry_invalidated:
```

Comment:
Read with diff of Strategy3

Ideally, I should surpress the Error only when `MODULE_UNINSTALL_FLAG`